stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -464,10 +477,39 @@ SpecialResult convertSpecial(const OpConversionPattern<VhloOpTy>& pattern,
     }
   }
   if (stablehloAttr) {
-    stablehloAttrs.emplace_back(
-        StringAttr::get(pattern.getContext(), stablehloName), stablehloAttr);
+    stablehloAttrs.emplace_back(stablehloName, stablehloAttr);
     return specialSuccess();
   }
+  if constexpr (std::is_same<VhloOpTy, vhlo::FftOpV1>::value) {
```

Comment:
A bit more overhead on this conversion since we are ambiguously converting TensorV1 to DenseElements and DenseArray, but not too bad. Overall worth it for compatibility.